jans-linux-setup/jans_setup/setup_app/utils/package_utils.py

```
@@ -127,17 +129,21 @@ def check_and_install_packages(self):
                 if install[install_type]:
                     self.logIt("Installing packages " + packages)
                     print("Installing packages", packages)
-                    if not base.os_type in ('fedora', 'suse'):
+                    if not base.os_type in ('suse',):
```

Ruff (0.15.9)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)